snapcraft/yaml.py

```diff
@@ -345,3 +351,35 @@ def _expand_filesets_for(stage, properties):
             new_stage_set.append(item)
 
     return new_stage_set
+
+
+def load_config():
+    global _config
+    if _config:
+        return _config
+
+    try:
+        _config = Config()
+        return _config
+    except SnapcraftYamlFileError as e:
+        logger.error(
+            'Could not find {}.  Are you sure you are in the right '
+            'directory?\nTo start a new project, use \'snapcraft '
+            'init\''.format(e.file))
+        sys.exit(1)
+    except SnapcraftSchemaError as e:
+        msg = 'Issues while validating snapcraft.yaml: {}'.format(e.message)
```

Comment:
Just an inconsistency: `msg` isn't created in any of these other cases-- the string is passed directly to `logger.error()`. You can safely ignore this, I just wanted to point it out.